src/webnn_native/dml/GraphDML.cpp

```diff
@@ -1102,6 +1102,34 @@ namespace webnn_native { namespace dml {
         return {};
     }
 
+    MaybeError Graph::AddSplit(const op::Split* split) {
+        DAWN_ASSERT(split->Inputs().size() == 1);
+        const OperandBase* inputOperand = split->Inputs()[0].Get();
+        DAWN_ASSERT(mExpression.find(inputOperand) != mExpression.end());
+        ::dml::Expression input = mExpression.at(inputOperand);
+        ::dml::TensorDimensions inputDims = input.GetOutputDesc().sizes;
+
+        // dml::Span just holds the refernces, need a variable to hold the memory.
+        std::vector<uint32_t> splits = split->GetSplits();
+        int32_t axis = split->GetAxis();
+        if (splits.size() == 1) {
+            if (inputDims[axis] % splits[0] != 0) {
+                return DAWN_INTERNAL_ERROR("the axis " + std::to_string(axis) + " with size " +
+                                           std::to_string(splits.size()) +
+                                           " can't be divisible by splits[0] " +
+                                           std::to_string(splits[0]) + ".");
+            }
+            splits = std::vector(splits[0], inputDims[axis] / splits[0]);
+        }
+        ::dml::Span<const uint32_t> splitsSpan(splits);
+        std::vector<::dml::Expression> output = ::dml::Split(input, axis, splitsSpan);
+        size_t outputSize = split->Outputs().size();
```

Comment:
Also need check output.size() == outputSize? Thanks!

Reply:
I think it's a good code style, i prefer adding the assert, done, thanks.

Reply:
Got it and we may need to adding the different asserts based on different actual cases.
LGTM, thanks! @fujunwei